source/uwp/SharedRenderer/lib/XamlHelpers.cpp

```diff
@@ -268,21 +283,98 @@ namespace AdaptiveCards::Rendering::Xaml_Rendering::XamlHelpers
         }
     }
 
+    winrt::fire_and_forget ConfigureSvgImageSourceAsync(winrt::AdaptiveRenderContext renderContext,
+                                                        winrt::hstring url,
+                                                        winrt::Uri uri,
+                                                        winrt::SvgImageSource svgImageSource,
+                                                        winrt::TileControl tileControl)
+    {
+        auto imageUrl = GetUrlFromString(renderContext.HostConfig(), url);
```

Comment:
can you pass in Url url as url has been already created in the callee? and remove hostconfig and hstring url?